src/app/shared/components/fy-userlist/fy-userlist-modal/fy-userlist-modal.component.ts

```
@@ -186,22 +174,22 @@
     }
   }
 
-  filterSearchedEmployees(searchedEmployees: Partial<Employee>[], employees: Partial<Employee>[]) {
+  filterSearchedEmployees(searchedEmployees: Partial<Employee>[], employees: Partial<Employee>[]): Partial<Employee>[] {
     searchedEmployees = searchedEmployees.filter(
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)